sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapi/AADResourceServerConfiguration.java

```diff
@@ -36,31 +38,39 @@
  */
 @Configuration(proxyBeanMethods = false)
 @ConditionalOnResource(resources = "classpath:aad.enable.config")
-@EnableConfigurationProperties({ AADAuthenticationProperties.class })
```

Comment:
This format is correct, the words between {} should have space before and after them. Maybe you should check your style file